docs/requirements.txt

```diff
@@ -0,0 +1,7 @@
+sphinx>=1.4.0,<=1.6.9
+sphinx_rtd_theme
+sphinxcontrib-pandoc-markdown
+ipython
+ipykernel
+nbsphinx
+breathe
```

Comment:
What is this file actually specifying here and who/what is using it?

Reply:
readthedocs needs this file to know what to install on their end.